api/layer/multipart_upload.go

```diff
@@ -1287,26 +1325,13 @@ func (n *layer) uploadPartAsSlot(ctx context.Context, params uploadPartAsSlotPar
 		Attributes:   params.attributes,
 		CreationTime: params.creationTime,
 		CopiesNumber: params.multipartInfo.CopiesNumber,
-		Multipart: &Multipart{
-			MultipartHashes: objHashes,
-		},
-	}
-
-	if params.uploadPartParams.Size > n.neoFS.MaxObjectSize()/2 {
-		chunk = n.buffers.Get().(*[]byte)
-		isReturnToPool = true
-	} else {
-		smallChunk := make([]byte, params.uploadPartParams.Size)
-		chunk = &smallChunk
-	}
-
-	id, elements, err = n.manualSlice(ctx, params.bktInfo, prm, splitFirstID, splitPreviousID, *chunk, params.payloadReader)
-	if isReturnToPool {
-		n.buffers.Put(chunk)
+		Payload:      params.payloadReader,
+		PayloadSize:  uint64(params.decSize),
 	}
 
+	id, objHashBts, err := n.objectPutAndHash(ctx, prm, params.bktInfo)
```

Comment:
So, no links to first ID?

Reply:
No, we upload Slot just as a regular object